mmhuman3d/models/registrants/smplify.py

```diff
@@ -783,5 +774,43 @@ def _expand_betas(self, batch_size, betas):
         return betas_video
 
     @staticmethod
-    def _compute_relative_loss_change(pre_v, cur_v):
+    def _compute_relative_change(pre_v, cur_v):
+        """Compute relative loss change. If relative change is small enough, we
+        can apply early stop to accelerate the optimization.
+
+        Args:
+            pre_v: previous value
+            cur_v: current value
+
+        Returns:
+            float: relative change
+        """
         return (pre_v - cur_v) / max([np.abs(pre_v), np.abs(cur_v), 1])
+
+    @staticmethod
+    def _skip_loss(loss, loss_weight_override):
+        """Whether to skip loss computation. In the table below, if the return
+        value is True, it means the computation of loss can be skipped. As the
+        result is 0 even if it is calculated, we can skip it to save
+        computational cost.
+
+        | loss.loss_weight | loss_weight_override | returns |
+        | ---------------- | -------------------- | ------- |
+        |      == 0        |         None         |   True  |
+        |      != 0        |         None         |   False |
+        |      == 0        |         == 0         |   True  |
+        |      != 0        |         == 0         |   True  |
+        |      == 0        |         != 0         |   False |
+        |      != 0        |         != 0         |   False |
+
+        Args:
+            loss: loss.loss_weight is assigned when loss is initialized
```

Comment:
We may explain here `loss` is an object that has attribute `loss_weight`.

Reply:
Revised in https://github.com/open-mmlab/mmhuman3d/pull/146/commits/e264b18059304b9e47229492bdb78f6c6cfcbde6